Documentation/ApiOverview/Backend/BackendModules/_DocHeaderComponent.rst.txt

```diff
@@ -0,0 +1,45 @@
+..  Generated by https://github.com/TYPO3-Documentation/t3docs-codesnippets
+..  php:namespace::  TYPO3\CMS\Backend\Template\Components
```

Comment:
```suggestion
..  php:namespace:: TYPO3\CMS\Backend\Template\Components
```

Reply:
Ups sorry. Just see: generated

Reply:
Yeah, needs to be fixed in t3docs/codesnippets then